samples/nrf_rpc/protocols_serialization/server/README.rst

```diff
@@ -48,7 +48,7 @@ The following snippets are available:
 User interface
 **************
 
-Button 1:
+Button 3 on nRF54L15pdk, Button 4 on nRF52840dk:
    * When the ``log_rpc`` snippet is enabled: triggers a fatal error.
      This is used for testing the crash log feature.
    * Otherwise: not available.
```

Comment:
When there are alternatives for these, put them in tabs.
```suggestion
.. tabs::

   .. group-tab:: nRF52840 DK
   
      Button 4:
         * When the ``log_rpc`` snippet is enabled: triggers a fatal error.
            This is used for testing the crash log feature.
         * Otherwise: not available.
         
   .. group-tab:: nRF54L15 DK

      Button 3:
         * When the ``log_rpc`` snippet is enabled: triggers a fatal error.
           This is used for testing the crash log feature.
         * Otherwise: not available.
```
Also noticed that the tab name on Line 116 should have a capital L for the tab name. This ensures that the correct tab is selected both places when the reader selects one.

Reply:
Fixed both.